app/frontend/src/features/communities/discussions/DiscussionPage.tsx

```diff
@@ -0,0 +1,130 @@
+
+const useStyles = makeStyles((theme) => ({
+  header: {
+    alignItems: "center",
+    display: "flex",
+  },
+  discussionTitle: {
+    marginInlineStart: theme.spacing(2),
+  },
+  discussionContent: {
+    margin: 0,
+  },
+  addedByLabel: {
+    marginBlockStart: theme.spacing(2),
+    marginBlockEnd: theme.spacing(1),
+  },
+  creatorContainer: {
+    "& > * + *": {
+      marginInlineStart: theme.spacing(2),
+    },
+    alignItems: "center",
+    display: "flex",
+    marginBlockEnd: theme.spacing(2),
+  },
+  creatorDetailsContainer: {
+    display: "flex",
+    flexDirection: "column",
+  },
+  avatar: {
+    height: "3rem",
+    width: "3rem",
+  },
+}));
+
+export const CREATOR_LOADING_TEST_ID = "creator-loading-state";
+
+export default function DiscussionPage() {
+  const classes = useStyles();
+  const { discussionId } = useParams<{
+    discussionId: string;
+    discussionSlug?: string;
+  }>();
+  const history = useHistory();
+
+  const { data: discussion, error, isLoading: isDiscussionLoading } = useQuery<
+    Discussion.AsObject,
+    GrpcError
+  >({
+    queryKey: discussionKey(+discussionId),
+    queryFn: () => service.discussions.getDiscussion(+discussionId),
+  });
+
+  const { data: discussionCreator, isLoading: isCreatorLoading } = useUser(
+    discussion?.creatorUserId
+  );
+
+  return (
+    <>
+      {error && <Alert severity="error">{error.message}</Alert>}
+      {isDiscussionLoading ? (
+        <CircularProgress />
+      ) : (
+        discussion && (
+          <>
+            <div className={classes.header}>
+              <HeaderButton
+                onClick={() => history.goBack()}
+                aria-label={PREVIOUS_PAGE}
+              >
+                <BackIcon />
+              </HeaderButton>
+              <PageTitle className={classes.discussionTitle}>
+                {discussion.title}
+              </PageTitle>
+            </div>
+            <Divider />
+            <Markdown source={discussion.content} />
+            <Typography className={classes.addedByLabel} variant="body1">
+              {ADDED_BY}
+            </Typography>
+            <div className={classes.creatorContainer}>
+              <Avatar
+                user={discussionCreator}
+                className={classes.avatar}
+                isProfileLink={false}
+              />
+              <div className={classes.creatorDetailsContainer}>
+                {isCreatorLoading ? (
+                  <Skeleton data-testid={CREATOR_LOADING_TEST_ID} />
+                ) : (
+                  <Typography variant="body1">
+                    {discussionCreator?.name ?? UNKNOWN_USER}
+                  </Typography>
+                )}
+                <Typography variant="body2">
+                  Created at{" "}
+                  {dateFormatter.format(timestamp2Date(discussion.created!))}
+                </Typography>
+              </div>
+            </div>
+            <Divider />
```

Comment:
I know there's no mockup for this, but IMO, don't use any dividers and put the user at the top under the header (without the "Added by" text)

Reply:
Yeah the trickiest part of all this has been figuring out the best way to lay this out without things looking terrible at the same time! I based it off the design for guides (also where I got the divider ideas off) which has the creator at the bottom. Do you think it's worth raising that to design and maybe it's better to have it near the top? Or are guides different enough that the current design as it is makes sense?